styles/aboutus.css

```diff
@@ -0,0 +1,164 @@
+:root {
+    --background-color: #FAF4E8;
+    --feature-color: #34959E;
+    --footer-color: #9D6034;
+    --nav-bar: #DAA46D;
+    --text-color: #382B22;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    background-color: var(--background-color);
+    font-family:Arial, Helvetica, sans-serif;
+    height: 90%;
+    margin: 0;
+    padding: 0;
+    width: 100%;
+}
+
+h2 {
+    margin: 0;
+}
+
+h3 {
+    margin: 0;
+}
+p {
+    margin: 0;
+}
+
+.header {
+    
+}
+
+.nav-bar {
+    /* margin: 32px 0 0 32px; */
+    background-color: #313640;
+    background-color: var(--nav-bar);
+    border-radius: 25px 25px 0 0;
+    display: flex;
+    flex-direction: column;
+    overflow-x: hidden;
+    overflow-y: auto; 
+    padding-top: 32px;
+    width: 240px;
+}
+
+.main {
+    /* width: 100%; */
+    background-color: var(--color-background);
+    display: flex;
+    gap: 32px;
+    height: calc(100% - 350px);
+    margin: 150px 32px 32px 32px;
+}
+
+
+.team-tabs {
+    /* gap: 18px; */
+    display: flex;
+    flex-direction: column;
+    gap: 32px;
+}
+
+    .tab {
+           
```

Comment:
This class is lacking CSS, maybe remove this unless you forgot to add CSS to it.